src/rate/interfaces/get-exchange-rate.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface IGetExchangeRate {
+  r030: number;
+  txt: string;
+  rate: number;
+  cc: string;
+  exchangedate: string;
+}
```

Comment:
Can you explain pls what do you solve with this Interface in you code ?

Reply:
this interface is used as data contract for function that calls NBU API to obtain information about exchange rate